docs/interface.md

```diff
@@ -30,9 +30,9 @@ partial interface WorkerNavigator {
 };
 
 dictionary ShareData {
-  DOMString? title;
-  DOMString? text;
-  DOMString? url;
+  DOMString title;
+  DOMString text;
+  USVString url;
```

Comment:
URLs need to use UTF-8 encode. Distinction between JavaScript strings (DOMString) and scalar value strings (USVString) is best explained here: https://infra.spec.whatwg.org/#strings. (IDL will be updated to use Infra in due course.)